app/src/main/java/ar/scacchipa/jsonplaceholderviewer/domain/MiddleCommentsUserCase.kt

```diff
@@ -0,0 +1,17 @@
+package ar.scacchipa.jsonplaceholderviewer.domain
+
+import ar.scacchipa.jsonplaceholderviewer.data.Comment
+import ar.scacchipa.jsonplaceholderviewer.data.ICommentRepository
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+
+class MiddleCommentsUserCase(
+    private val jphRepository: ICommentRepository// = JPHRepository()
```

Comment:
nice name for the Interface but maybe we can put a better name on the implementation, like CommentResposityImpl or CommentResposity_Impl just to put some examples.